docs/src/how-to/index.rst

```diff
@@ -14,4 +14,5 @@ are a total beginner, you can go to :ref:`userdoc-get-started` section.
     sample-selection
     property-selection
     keys-selection
+    le-basis
```

Comment:
Should we put this after the `splined-radial-integral` example?

Reply:
I feel like the splined radial integral is something that people won't use much. Let's keep it last

Reply:
fair

Reply:
Agreed on keeping it last. It might be worth to come back to this example and update it with another basis (maybe a bunch of equally spaced gaussians?) now that we have nicer ways to handle the LE basis.